main.js

```diff
@@ -43,29 +43,29 @@ class Car {
     this.$img.style = 'top:' + this.location[1] + 'px ; left:' + this.location[0] + 'px'
   }
   start() {
-    setInt = setInterval(() => this.move(), 16)
+    this.interval = setInterval(() => this.move(), 16)
+    alfaRomeo.isRunning = true
```

Comment:
You don't want to reference a specific _instance_ of your class. Use `this` instead.